### DISCUSSION TITLE
3 finger swipe

### DISCUSSION THREAD

**Discussion by Notaduck** [nikitabobko/AeroSpace#876] [questions-and-answers]
Hi all of you, do anybody knows if it is possible to do a 3 finger swipe to switch between workspaces in Aerospace?

**Comment by filipw01** [comment]
I haven't done it, but it's mentioned in the [goodies](https://nikitabobko.github.io/AeroSpace/goodies#use-trackpad-gestures-to-switch-workspaces)
